swagger.yml

```diff
@@ -2025,8 +2049,8 @@ paths:
           $ref: '#/components/responses/error'
   /alerts:
     get:
-      summary: search alerts using the filters in the request
-      description: search alerts using the filters in the request
+      summary: search alerts
+      description: 'This API fetches all alerts. The parameters in version 2.0 are not used since all filters are executed by the frontend. This API is also called by the Dashboard. '
```

Comment:
Is "on the frontend side" better than "by the frontend"?

Reply:
this is not an error !